### PR TITLE
Math magicians: components.

### PR DESCRIPTION
**Pull request by tiagorahal**
### Math magicians: components
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component
- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components.
- Once the Calculator is ready, import it and use it in your main component (App.js).

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,19 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Calculator from './components/Calculator';
```
Comment (fernandorpm) [inline src/App.js]: Change the imported file name `./components/Calculator` to `./components/calculator`, to match the actual file name: `calculator.js`, or change the file to `Calculator.js`

**Note the capitalized C!**

Cheers! 🚀

**Review by fernandorpm — CHANGES_REQUESTED** [review]

File: src/components/calculator.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../App.css';
+
```
Comment (arthurborgesdev) [inline src/components/calculator.js:3]: This is the file that you have to capitalize the name.

File: src/App.js (hunk 1 above)
Comment (tiagorahal) [reply src/App.js]: Thank you for the suggested changes.

File: src/components/calculator.js (hunk 2 above)
Comment (tiagorahal) [reply src/components/calculator.js:3]: Thank you for the suggested changes.

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi Team 😃 , 
## STATUS: APPROVED 👍 
Great job implementing all the requirements for this milestone!! 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**